Sources/Apodini/SyntaxTreeVisitor/DelegatingHandlerInitializerVisitor.swift

```diff
@@ -44,7 +44,11 @@ class DelegatingHandlerInitializerVisitor: HandlerVisitor {
             entry.collectMetadata(self.visitor)
         }
 
-        handler.metadata.collectMetadata(self.visitor)
+        print("We found metadata: \(metadata) for \(H.self)")
```

Comment:
Please remove or use the Application‘s logger!

Reply:
Good catch. Was some debug printing. Was removed.